holidays/holiday_base.py

```diff
@@ -1026,7 +1026,7 @@ def pop(self, key: DateLike, default: Union[str, Any] = None) -> Union[str, Any]
 
         return dict.pop(self, self.__keytransform__(key), default)
 
-    def pop_named(self, name: str) -> list[date]:
+    def pop_named(self, name: str, lookup: str = "icontains") -> list[date]:
```

Comment:
Could you update the method's documentation with the new param too?

Reply:
Changed the method's docstring